src/release-notify-action.js

```diff
@@ -24,12 +22,25 @@ async function runAction(token, staleDays, commitMessageLines) {
   - author:${latestRelease.author.login}
 `)
 
+  const closedNotifyIssues = await tryGetClosedNotifyIssues(
+    token,
+    latestRelease.published_at
+  )
+
+  // exit if there is a closed notify issue but stale days have not passed
+  if (
+    closedNotifyIssues?.length &&
+    !isClosedNotifyIssueStale(closedNotifyIssues, staleDate)
```

Comment:
once you do the change above, this condition becomes simply:

if(!closedNotifyIssue)

and you can even rename it to something like `closedSnoozedIssue` or something like that, to convey the meaning that this is an issue which was closed as not planned and within the snooze period